tests/OSPSuite.HelpersForTests/ModelHelperForSpecs.cs

```diff
@@ -103,6 +103,40 @@ private IndividualBuildingBlock getIndividual()
             Value = 10,
             Dimension = amountPerTimeDimension
          });
+
+         individual.Add(new IndividualParameter
+         {
+            //new parameter that does not exist
+            Path = _objectPathFactory.CreateObjectPathFrom(Organism, ArterialBlood, "NEW_PARAM_DISTRIBUTED"),
+            DistributionType = DistributionType.Normal,
```

Comment:
example here of a dsitrbitued parameter with required sub parameters (MEAN, DEVIATION and PERCENTILE)